### PR TITLE
fix(defaults): properly pass count to quickfix commands

### LINKED ISSUE #30626: `[Q` failed on count with `Wrong number of arguments`
### Problem

With the newly unimpaired keybinding, `[Q` will show error if used with count.

```
E5108: Error executing lua: vim/_defaults.lua:0: Wrong number of arguments
stack traceback:
	[C]: in function 'crewind'
	vim/_defaults.lua: in function <vim/_defaults.lua:0>
```

### Steps to reproduce

nvim --clean
:grep <something>
1[Q

### Expected behavior

Should behave the same as if `:crewind <count>` is run

### Nvim version (nvim -v)

v0.11.0-dev-876+g5331f87f61

### Vim (not Nvim) behaves the same?

N/A

### Operating system/version

Arch linux 6.11.1-arch1-1

### Terminal name/version

foot v1.18.1

### $TERM environment variable

foot

### Installation

AUR

### PR DESCRIPTION
**Pull request by zeertzjq**
Fix #30626

### CODE REVIEW

**Review by justinmk — APPROVED** [review]

File: runtime/lua/vim/_defaults.lua [hunk 1]
```diff
@@ -225,13 +225,13 @@ do
     })
 
     vim.keymap.set('n', '[Q', function()
-      vim.cmd.crewind({ args = vim.v.count ~= 0 and { vim.v.count } or nil })
+      vim.cmd.crewind({ count = vim.v.count ~= 0 and vim.v.count or nil })
```
Comment (gpanders) [inline runtime/lua/vim/_defaults.lua:228]: `:h :crewind` says:

```
:cr[ewind][!] [nr]	Display error [nr].  If [nr] is omitted, the FIRST
			error is displayed.  See |:cc|.
```

so why does `args` not work here? `:crewind 1` works as expected

Comment (gpanders) [reply runtime/lua/vim/_defaults.lua:228]: `count` has the same effect (this isn't obvious to me from the help description) so this is fine.

Comment (jamestansx) [reply runtime/lua/vim/_defaults.lua:228]: `vim.api.nvim_parse_cmd("crewind 123", {})`  do actually parse the argument to `count` 

```
{                                                                                    
  addr = "?",                                                                                  
  args = {},                                                                                   
  bang = false,                                                                                
  cmd = "crewind",                                                                             
  count = 123,                                                                                 
  magic = {                                                                                    
    bar = true,                                                                                
    file = false                                                                               
  },                                                                                           
  mods = {                                                                                     
    browse = false,                                                                            
    confirm = false,                                                                           
    emsg_silent = false,                                                                       
    filter = {                                                                                 
      force = false,                                                                           
      pattern = ""                                                                             
    },                                                                                         
    hide = false,                                                                              
    horizontal = false,                                                                        
    keepalt = false,                                                                           
    keepjumps = false,                                                                         
    keepmarks = false,                                                                         
    keeppatterns = false,                                                                      
    lockmarks = false,                                                                         
    noautocmd = false,                                                                         
    noswapfile = false,                                                                        
    sandbox = false,                                                                           
    silent = false,                                                                            
    split = "",                                                                                
    tab = -1,                                                                                  
    unsilent = false,                                                                          
    verbose = -1,                                                                              
    vertical = false                                                                           
  },                                                                                           
  nargs = "0",                                                                                 
  nextcmd = "",                                                                                
  range = { 123 }                                                                              
}   
```

Comment (zeertzjq) [reply runtime/lua/vim/_defaults.lua:228]: It was #23121

**Pull request merged**